File: VB/WebSolution2.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl

Namespace WebSolution2.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim sam As SimpleUser = ObjectSpace.FindObject(Of SimpleUser)(New BinaryOperator("UserName", "Sam"))
			If sam Is Nothing Then
				sam = ObjectSpace.CreateObject(Of SimpleUser)()
				sam.UserName = "Sam"
				sam.IsAdministrator = True
				sam.Save()
			End If
			Dim john As SimpleUser = ObjectSpace.FindObject(Of SimpleUser)(New BinaryOperator("UserName", "John"))
			If john Is Nothing Then
				john = ObjectSpace.CreateObject(Of SimpleUser)()
				john.UserName = "John"
				john.IsAdministrator = True
				john.Save()
			End If
		End Sub
	End Class
End Namespace

File: CS/WebSolution2.Module/Updater.cs
using System;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;

namespace WebSolution2.Module {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            SimpleUser sam = ObjectSpace.FindObject<SimpleUser>(new BinaryOperator("UserName", "Sam"));
            if (sam == null) {
                sam = ObjectSpace.CreateObject<SimpleUser>();
                sam.UserName = "Sam";
                sam.IsAdministrator = true;
                sam.Save();
            }
            SimpleUser john = ObjectSpace.FindObject<SimpleUser>(new BinaryOperator("UserName", "John"));
            if (john == null) {
                john = ObjectSpace.CreateObject<SimpleUser>();
                john.UserName = "John";
                john.IsAdministrator = true;
                john.Save();
            }
        }
    }
}
